src/core/adaptors/signUp/index.adaptors.ts

```diff
@@ -16,15 +17,54 @@ export const preRegister = async (email: string): Promise<AdaptorRes<PreRegister
     };
   }
 };
+export const registerAdaptor = async (email: string, password?: string): Promise<AdaptorRes<SuccessRes>> => {
+  try {
+    const res = await register({ email, password });
+    if (res.message === 'success')
```

Comment:
FMI: do you have another situation in which our API returns 200 but does not succeed?

Reply:
for other APIs too